lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -89,11 +82,14 @@ class ArtworkRail extends React.Component {
   }
 
   mainContainerHeight() {
+    // includes height of View All button (if present), the separator, and a 30px margin
+    const supplementaryHeight = this.hasAdditionalContent() ? 101 : 30
+
     if (this.expandable()) {
       const initialRailHeight = isPad ? 460 : 530
```

Comment:
We should totally (down the road) move these to stylesheets for e.g. ‘regular’ and ‘compact’ size classes, like Apple does with `UIUserInterfaceSizeClass`.